dataverse-webapp/src/main/webapp/resources/vecler/theme.scss

```diff
@@ -1205,6 +1211,14 @@ tr.ui-state-highlight a:not(.btn), .ui-widget-content tr.ui-state-highlight a:no
 	width: 8em;
 }
 
+#form\:undoLastReorder {
```

Comment:
It would be better to make this selector different. something like adding class to panel in xhtml. and use it like: `.undo-last-reorder-panel`. Right now it is very fragile to changes because it depends on 2 id's of elements and correlation between them.

Reply:
Done